### PR TITLE
Tw 368 Add APR label to Youves tokens

### PR DESCRIPTION
**Pull request by svjatoslavtt**

### CODE REVIEW

File: src/utils/get-delegate-text.util.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { youvesTokens } from '../apis/youves/constants';
+import { TokenInterface } from '../token/interfaces/token.interface';
+
+export const getDelegateText = (token: TokenInterface) =>
+  (youvesTokens as string[]).includes(token.symbol) ? 'APR' : 'APY';
```
Comment (herkoss) [inline src/utils/get-delegate-text.util.ts:5]: Remove this assertion, please

**Review by herkoss — COMMENTED** [review]
Well done!
Let's improve this solution!)

**Review by unixvb — DISMISSED** [review]

File: src/apis/youves/index.ts [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+export const getYOUTokenApr$ = (
+  assetToUsdExchangeRate: BigNumber,
+  governanceToUsdExchangeRate: BigNumber
+): Observable<number> => {
+  const unifiedStaking = new UnifiedStaking(toolkit, indexerConfig, mainnetNetworkConstants);
+
+  return from(unifiedStaking.getAPR(assetToUsdExchangeRate, governanceToUsdExchangeRate)).pipe(
+    map(value => Number(value.multipliedBy(MULTIPLIER))),
+    catchError(error => {
+      console.log('Youves error: get YOU token APR', error);
```
Comment (herkoss) [inline src/apis/youves/index.ts:46]: Delete this `console.log`

File: src/apis/youves/index.ts [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+      return of(0);
```
Comment (herkoss) [inline src/apis/youves/index.ts:48]: Fix magic numbers. `DEFAULT_APR` for example

**Review by herkoss — COMMENTED** [review]
Well done!

File: src/apis/youves/index.ts (hunk 2 above)
Comment (svjatoslavtt) [reply src/apis/youves/index.ts:46]: why?

**Review by herkoss — DISMISSED** [review]

File: package.json [hunk 4]
```diff
@@ -179,6 +183,12 @@
...
     "shelljs": "^0.8.5",
     "simple-plist": "^1.3.1",
-    "moment": "^2.29.2"
+    "moment": "^2.29.2",
+    "@taquito/taquito": "15.0.1",
+    "@taquito/utils": "15.0.1",
+    "@taquito/beacon-wallet": "15.0.1",
+    "@taquito/contracts-library": "15.0.1",
+    "@taquito/tzip16": "15.0.1",
+    "bignumber.js": "9.1.1"
```
Comment (herkoss) [inline package.json:192]: Why do we need these dev deps?

File: src/components/token-screen-content-container/token-header.tsx [hunk 5]
```diff
@@ -27,6 +30,8 @@ interface Props {
   token: TokenInterface;
 }
 
+const DECIMAL_VALUE = 2;
```
Comment (herkoss) [inline src/components/token-screen-content-container/token-header.tsx:33]: The same in `DelegateTag`. Move to constants, please

**Review by herkoss — COMMENTED** [review]
GJ!

File: package.json (hunk 4 above)
Comment (svjatoslavtt) [reply package.json:192]: because we have conflicts of version

**Review by herkoss — DISMISSED** [review]

**Review by Boltman92 — APPROVED** [review]

**Review by herkoss — APPROVED** [review]

**Review by Boltman92 — DISMISSED** [review]

**Review by NovakVlad2424 — DISMISSED** [review]
done ios, android: 1.15_751

File: babel.config.js [hunk 6]
```diff
@@ -1,6 +1,6 @@
 module.exports = {
   presets: ['module:metro-react-native-babel-preset'],
-  plugins: ['react-native-reanimated/plugin'],
+  plugins: ['react-native-reanimated/plugin', ['@babel/plugin-proposal-decorators', { legacy: true }]],
```
Comment (lourenc) [inline babel.config.js:3]: It's an unnecessary change

File: package.json [hunk 7]
```diff
@@ -30,6 +30,7 @@
   "dependencies": {
     "@airgap/beacon-sdk": "3.1.4",
     "@apollo/client": "^3.7.1",
+    "@babel/plugin-proposal-decorators": "^7.21.0",
```
Comment (lourenc) [inline package.json:33]: Please, remove the decorators support. It's not needed at all.

**Review by lourenc — CHANGES_REQUESTED** [review]

**Pull request merged**